src/seeds/access.ts

```diff
@@ -38,17 +39,30 @@ export const restrictAccess = (payload: MessagePayload, signer?: EthereumAddress
   }
 }
 
-export const ownerOrAdmin = (payload: MessagePayload, signer?: EthereumAddress) => {
+export const ownerOrAdmin = async (payload: MessagePayload, signer: EthereumAddress, dbClient: DBClient) => {
   if (!signer) {
     throw new Error('must specify a signer');
   }
 
   if (!notAdminAddress(signer)) {
```

Comment:
double not is usually bad for readability lol - maybe change the function to isAdminAddress ?